exercises/concept/lasagna/.meta/exemplar.cpp

```diff
@@ -0,0 +1,24 @@
+// ovenTime returns the amount in minutes that the lasagna should stay in the
+// oven.
+int ovenTime() { return 40; }
```

Comment:
No, this is a function, that retunrs an integer. It could be switched to a constant like this:
```cpp
const int OVEN_TIME{40};
```
But I feel that this is not in the scope of the first exercise, so we use a function here.